draft-ietf-jsonpath-base.md

```diff
@@ -1510,15 +1510,16 @@ exponential, CPU usage.
 ## Attacks on Security Mechanisms that Employ JSONPath
 
 Where JSONPath is used as a part of a security mechanism, attackers
-can attempt to evoke unexpected behavior, or take advantage of
-differences in behavior between JSONPath implementations.
-
-This also applies to underlying technologies such as UTF-8 (see
-{{Section 10 of -utf8}}), the Unicode character set, and JSON.
-A characteristic of JSON that can lead to varying results is the
-fact that JSON objects are unordered; therefore, the order in which
-results of a JSONPath query reflect the presence of JSON object
-members can vary with implementations.
+can attempt to provoke unexpected or unpredictable behavior, or
+take advantage of differences in behavior between JSONPath implementations.
+
+Unexpected or unpredictable behaviour can arise from an argument with certain
+constructs described as unpredictable by {{-json}}.
```

Comment:
Note that I didn't adopt the language of https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base/issues/186 to describe certain constructs as "deprecated but allowed" by RFC 8259 since that RFC does not use the term deprecation.